src/components/Gradebook/GradebookFilters/AssignmentFilter/test.jsx

```diff
@@ -9,12 +10,20 @@ import {
   mapDispatchToProps,
 } from '.';
 
-jest.mock('data/selectors/filters', () => ({
+jest.mock('data/selectors', () => ({
   /** Mocking to use passed state for validation purposes */
-  selectableAssignmentLabels: jest.fn().mockImplementation((state) => ({
-    state,
-    selectableLabels: 'selectableLabels',
-  })),
+  filters: {
```

Comment:
These values are arbitrary but should differ from each other to catch bad linking of filters to props

Reply:
++